personality/argus/config.pan

```diff
@@ -99,79 +108,82 @@ variable ARGUS_SERVICES = {
-  contents = contents + '#   chkconfig: 345 97 97' + "\n";
-  contents = contents + '#' + "\n";
-  contents = contents + '#   description:  Argus services startup script' + "\n";
-  contents = contents + '#' + "\n";
-  contents = contents + '#   Version: V1.5' + "\n";
-  contents = contents + '#' + "\n";
-  contents = contents + '#   Date: 15/10/2012' + "\n";
-  contents = contents + '###############################################################################' + "\n";
-  foreach (service;serviceinfo;ARGUS_SERVICES) {
-    contents = contents + service + '_HOME=' + serviceinfo['home'] + "\n";
-    contents = contents + 'export ' + service + '_HOME' + "\n";
-  };
-  contents = contents + "\n";
-  contents = contents + 'if [ `id -u` -ne 0 ]; then' + "\n";
-  contents = contents + '    echo "You need root privileges to run this script"' + "\n";
-  contents = contents + '    exit 1' + "\n";
-  contents = contents + 'fi ' + "\n";
-  contents = contents + "\n";
-  contents = contents + 'case "$1" in' + "\n";
-  contents = contents + '    start)' + "\n";
-  foreach (service;serviceinfo;ARGUS_SERVICES) {
-    contents = contents + '        echo "##### ' + service + ' #####"' + "\n";
-    contents = contents + '        ' + serviceinfo['initScript'] + ' status 2&> /dev/null' + "\n";
-    contents = contents + '        result=$?' + "\n";
-    contents = contents + '        if [ $result -ne 0 ]; then' + "\n";
-    contents = contents + '            echo "Starting ' + service + ': "' + "\n";
-    contents = contents + '            ' + serviceinfo['initScript'] + ' start' + "\n";
-    contents = contents + '            sleep 5' + "\n";
-    contents = contents + '        else' + "\n";
-    contents = contents + '            echo "' + service + ' already running" 1>&2' + "\n";
-    contents = contents + '        fi' + "\n";
-  };
-  contents = contents + '        ;; ' + "\n";
-  contents = contents + '    stop)' + "\n";
-  foreach (service;serviceinfo;ARGUS_SERVICES) {
-    contents = contents + '        echo "##### ' + service + ' #####"' + "\n";
-    contents = contents + '        echo "Shutting down ' + service + ': "' + "\n";
-    contents = contents + '        ' + serviceinfo['initScript'] + ' stop' + "\n";
-  };
-  contents = contents + '        ;;' + "\n";
-  contents = contents + '    status)' + "\n";
-  foreach (service;serviceinfo;ARGUS_SERVICES) {
-    contents = contents + '        echo "##### ' + service + ' #####"' + "\n";
-    contents = contents + '        ' + serviceinfo['initScript'] + ' status' + "\n";
-  };
-  contents = contents + '        ;;' + "\n";
-  contents = contents + '    *)' + "\n";
-  contents = contents + '        echo "Usage: $0 {start|stop|status}"' + "\n";
-  contents = contents + '        exit 1' + "\n";
-  contents = contents + '        ;;' + "\n";
-  contents = contents + 'esac' + "\n";
-  contents = contents + "\n";
-  contents = contents + 'exit 0' + "\n";
-
-  contents;
+    contents = "#!/bin/bash\n";
```

Comment:
use a heredoc?

Reply:
Or `file_contents` and `replace`?

Reply:
or `file_contents` and `format` if there's not too much duplication.